server/container/guice/protocols/jmap/src/main/java/org/apache/james/jmap/draft/JMAPModule.java

```diff
@@ -108,6 +112,7 @@ protected void configure() {
         install(new JMAPCommonModule());
         install(new DraftMethodsModule());
         install(new RFC8621MethodsModule());
+        install(new CustomMethodModule());
```

Comment:
This binding should be done using the james server builder module override in a test

I suggest putting this in CustomMethodContract.scala